includes/functions.php

```diff
@@ -3119,6 +3119,97 @@ function pmpro_round_price( $price, $currency = '' ) {
 	return $rounded;
 }
 
+/**
+ * Rounds price based on currency and returns a string.
+ *
+ * Does not format price, to do that, call pmpro_formatPrice().
+ *
+ * @since TBD
+ *
+ * @param int|float|string $amount   The amount to get price information for.
+ * @param null|string      $currency The currency to use, defaults to current currency.
+ *
+ * @return string The rounded price as a string.
+ */
+function pmpro_round_price_as_string( $amount, $currency = null ) {
+	$price_info = pmpro_get_price_info( $amount, $currency );
+
+	if ( ! $price_info ) {
+		return (string) pmpro_round_price( $amount, $currency );
+	}
+
+	return $price_info['amount_string'];
+}
+
+/**
+ * Get the price information about the provided amount.
+ *
+ * @since TBD
+ *
+ * @param int|float|string $amount   The amount to get price information for.
+ * @param null|string      $currency The currency to use, defaults to current currency.
+ *
+ * @return array The price information about the provided amount.
+ */
+function pmpro_get_price_info( $amount, $currency = null ) {
+	if ( ! is_numeric( $amount ) ) {
+		return false;
+	}
+
+	$amount = (float) $amount;
+
+	$currency_info = pmpro_get_currency( $currency );
+
+	$price_info = [
+		// The amount represented as a float.
+		'amount'        => $amount,
+		// The flat amount represent (example: 1.99 would be 199).
+		'amount_flat'   => 0,
```

Comment:
```suggestion
		'amount_flat'   => 0,
		// The number of decimals for this currency.
		'decimals'        => (int) $currency_info['decimals'],
```

Reply:
What was the purpose of adding this here if we pass it in `$price_info['currency']` already?